protobuf/types/workflow.proto

```diff
@@ -56,11 +40,29 @@ message Workflow {
       string key = 1;               // Key of the mapping.
       repeated Output outputs = 2;  // Outputs of the mapping.
     }
+
+    message Filter {
```

Comment:
If `Filter` with contintion is here, then go `filter` package should be moved under workflow|scheducler directory.

Otherwise move message Filter out of workflow space.